diff --git a/ios/Helpers.swift b/ios/Helpers.swift
index 804cd9bd9535ec7b9792b4d044e4d81bbe253ab9..e3ea29e421ebbc21ee1a81e53de978ca87b0ed98 100644
--- a/ios/Helpers.swift
+++ b/ios/Helpers.swift
@@ -108,7 +108,7 @@ func sampleQueryAsync(
       sampleType: sampleType,
       predicate: predicate,
       limit: limit,
-      sortDescriptors: sortDescriptors,
+      sortDescriptors: sortDescriptors
     ) { (_: HKSampleQuery, samples: [HKSample]?, error: Error?) in
       DispatchQueue.main.async {
         if let error = error {
diff --git a/ios/MedicationModule.swift b/ios/MedicationModule.swift
index 834aa60c9b6da25741d76529508c0b89cd0fc0c6..51f951da84d5c1fee04b02ecf8b4c5faac6e6638 100644
--- a/ios/MedicationModule.swift
+++ b/ios/MedicationModule.swift
@@ -142,7 +142,7 @@ import NitroModules
         relatedCodings: medication.medication.relatedCodings.map({ c in
           return RelatedCoding(system: c.system, code: c.code, version: c.version)
         })
-      ),
+      )
     )
   }
 
diff --git a/ios/PredicateHelpers.swift b/ios/PredicateHelpers.swift
index f3c3d65bda9c47920f68173b9a161bb47a28a2e4..ee6363bf53f84f961a55b178353f9747ed700528 100644
--- a/ios/PredicateHelpers.swift
+++ b/ios/PredicateHelpers.swift
@@ -75,7 +75,7 @@ func getPredicateForWorkoutBase(_ filter: FilterForWorkouts?) -> FilterForWorkou
       uuid: filter.uuid,
       sources: filter.sources,
       uuids: filter.uuids,
-      date: filter.date,
+      date: filter.date
     )
   }
   return nil
